xrm/xrm.d.ts

```diff
@@ -487,7 +484,7 @@ declare namespace Xrm
         /**
          * Called when the operation is successful.
          */
-        export type SuccessCallbackDelegate = () => void;
+        export type SuccessCallbackDelegate = ( object?: any ) => void;
```

Comment:
This probably should just be `object: any`. Optional parameters are not useful for typing callbacks, since a callback function `() => {}` still satisfies the `(object: any) => void` type.